snippets/08_googlevideo.txt

```diff
@@ -17701,4 +17731,5 @@
 2a00:1450:401e:3f::9 r3---sn-4g5edn7z.googlevideo.com
 2a00:1450:401e:3f::a r4---sn-4g5edn7z.googlevideo.com
 2a00:1450:401e:3f::b r5---sn-4g5edn7z.googlevideo.com
-2a00:1450:401e:3f::c r6---sn-4g5edn7z.googlevideo.com
\ No newline at end of file
+2a00:1450:401e:3f::c r6---sn-4g5edn7z.googlevideo.com

```

Comment:
I added two blank line here to keep this snippet same as others. If not, in the final merged hosts, this line will directly be stuck with next line like this:
![image](https://cloud.githubusercontent.com/assets/8066567/17846417/cea6e656-687a-11e6-89a0-93e0246cbce1.png)

It seems that the latest version of hosts is not merged from the latest snippets.